.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "eslint.validate": ["astro"],
   "[astro]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.defaultFormatter": "astro-build.astro-vscode",
```

Comment:
Should we add a `.vscode/extensions.json` with a `recommendations` object so this extension is proposed to be installed automatically?

Reply:
Already there ;)